src/lib/utility_functions.js

```diff
@@ -97,6 +97,24 @@ function removeEmptyElementsFromArray(array) {
   return tempArray;
 }
 
+/*
+ * Decodes a URI and form-encoded string (encoded via 
+ * ' ' => '+'). All '+' characters will be replaced
+ * with a ' ' and the result will be URI-decoded.
+ *
+ * TODO: Add unit tests
+ *
+ * @param {*} encodedValue: A URI and form-encoded string
+ * @returns {*} A string that has been unencoded
+ *
+ */
+export function decodedQueryParameter(encodedValue) {
+  return decodeURIComponent(
+    encodedValue.replace(/\+/g, "%20")
+  );
+  // return new SearchTerms(termsString).asArray();
```

Comment:
Any reason to keep this old code hanging around?

Reply:
Oops, good catch, thanks.